asciidoctor-revealjs.gemspec

```diff
@@ -30,6 +30,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rake', '~> 10.4.2'
   s.add_development_dependency 'asciidoctor-doctest', '= 2.0.0.beta.4'
   s.add_development_dependency 'pry', '~> 0.10.4'
+  s.add_development_dependency 'pry-byebug'
```

Comment:
You can conditionally include it only for non-jruby engine.

Reply:
It's done